clustering/web/extension/src/main/resources/schema/distributable-web_1_0.xsd

```diff
@@ -20,6 +20,11 @@
                     <xs:documentation>Defines a deployment specific Infinispan cache-based session management provider</xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="hotrod-session-management" type="tns:hotrod-session-management">
+                <xs:annotation>
+                    <xs:documentation>Defines a deployment specific Infinispan cache-based session management provider</xs:documentation>
```

Comment:
Seems like this should be different from what's above.

Reply:
Yep.